app/grandchallenge/reader_studies/forms.py

```diff
@@ -563,7 +561,7 @@ def _get_default_field(self, *, interface, current_value):
 class DisplaySetUpdateForm(DisplaySetCreateForm):
     _possible_widgets = {
         SelectUploadWidget,
-        *DisplaySetCreateForm._possible_widgets,
+        *InterfaceFormField._possible_widgets,
```

Comment:
Why is this being replaced? `DisplaySetUpdateForm` subclasses `DisplaySetCreateForm` so can include its widgets.